scwx-qt/source/scwx/qt/manager/alert_manager.hpp

```diff
@@ -20,6 +22,7 @@ class AlertManager : public QObject
    explicit AlertManager();
    ~AlertManager();
 
+   void SetRadarSite(std::shared_ptr<config::RadarSite> radarSite);
```

Comment:
Parameter should take a const reference to avoid a possible second copy: `const std::shared_ptr<...>&`

I've been inconsistent myself, but have been cleaning them up as I touch them.